scoring/src/main/scala/ai/h2o/sparkling/ml/internals/H2OMetric.java

```diff
@@ -39,7 +39,11 @@
   MaxPerClassError(false),
   Accuracy(true),
   MSE(false),
-  RMSE(false);
+  RMSE(false),
+  Withinss(false),
+  Betweenss(true),
+  TotWithinss(false),
+  Totss(false);
```

Comment:
The boolean here means `higherTheBetter`. @maurever can you please double check this make sense?

If I undertand correctly, `totss` is a bit more tricky and could have value `true` and `false` so I just picked one

Reply:
I double-check and it looks correct. For `Totss` I found the less the better.

Reply:
Thank you!